src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/crud-table/crud-table.component.html

```diff
@@ -29,9 +29,8 @@
                         [tableActions]="meta.actions">
       </cd-table-actions>
     </div>
-    <ng-container [ngIf]="expandedRow && meta.detail_columns.length > 0"
-                  *cdTableDetail>
-      <table class="table table-striped table-bordered">
+    <ng-container *ngIf="expandedRow && meta.detail_columns.length > 0">
+      <table *cdTableDetail class="table table-striped table-bordered">
```

Comment:
there will be some html lint error here since they don't usually allow two attributes to be in a single line.

Reply:
fixed